src/options.cpp

```diff
@@ -1626,28 +1626,32 @@ void options_manager::show(bool ingame)
         g->mmenu_refresh_motd();
         g->mmenu_refresh_credits();
     }
-    if( used_tiles_changed ) {
 #ifdef TILES
+    if( used_tiles_changed ) {
```

Comment:
Turns out that moving this fully inside the `#ifdef` made the vars `used_tiles_changed` and `pixel_minimap_height_changed` be regarded as "set but not used" (`-Werror=unused-but-set-variable`) in all the curses builds except Linux32 (which builds with `clang++`, so presumably that makes the difference there).